video/api/Analyze/Program.cs

```diff
@@ -216,13 +224,225 @@ public static object TranscribeVideo(string uri)
+            var timeOffset = frame.TimeOffset;
+            Console.Write("Time offset for the first frame: ");
+            Console.WriteLine(timeOffset.Seconds + timeOffset.Nanos * 1e9);
+            Console.WriteLine("Rotated Bounding Box Vertices:");
+            foreach (var vertex in frame.RotatedBoundingBox.Vertices)
+            {
+                Console.WriteLine(
+                    $"\tVertex x: {vertex.X}, Vertex.y: {vertex.Y}");
+            }
+            return 0;
+        }
+        // [END video_detect_text_gcs]
+
+        // [START video_detect_text]
+        public static object DetectText(string filePath)
+        {
+            var client = VideoIntelligenceServiceClient.Create();
+            var request = new AnnotateVideoRequest
+            {
+                InputContent = Google.Protobuf.ByteString.CopyFrom(File.ReadAllBytes(filePath)),
+                Features = { Feature.TextDetection },
+            };
+
+            Console.WriteLine("\nProcessing video for text detection.");
+            var op = client.AnnotateVideo(request).PollUntilCompleted();
+
+            // Retrieve the first result because only one video was processed.
+            var annotationResults = op.Result.AnnotationResults[0];
+
+            // Get only the first result.
+            var textAnnotation = annotationResults.TextAnnotations[0];
+            Console.WriteLine($"\nText: {textAnnotation.Text}");
+
+            // Get the first text segment.
+            var textSegment = textAnnotation.Segments[0];
+            var startTime = textSegment.Segment.StartTimeOffset;
+            var endTime = textSegment.Segment.EndTimeOffset;
+            Console.Write(
+                $"Start time: {startTime.Seconds + startTime.Nanos / 1e9 }, ");
+            Console.WriteLine(
+                $"End time: {endTime.Seconds + endTime.Nanos / 1e9 }");
+
+            Console.WriteLine($"Confidence: {textSegment.Confidence}");
+
+            // Show the result for the first frame in this segment.
+            var frame = textSegment.Frames[0];
+            var timeOffset = frame.TimeOffset;
+            Console.Write("Time offset for the first frame: ");
+            Console.WriteLine(timeOffset.Seconds + timeOffset.Nanos * 1e9);
+            Console.WriteLine("Rotated Bounding Box Vertices:");
+            foreach (var vertex in frame.RotatedBoundingBox.Vertices)
+            {
+                Console.WriteLine(
+                    $"\tVertex x: {vertex.X}, Vertex.y: {vertex.Y}");
+            }
+            return 0;
+        }
+        // [END video_detect_text]
+
+        // [START video_object_tracking_gcs]
+        public static object TrackObjectGcs(string gcsUri)
+        {
+            var client = VideoIntelligenceServiceClient.Create();
+            var request = new AnnotateVideoRequest
+            {
+                InputUri = gcsUri,
+                Features = { Feature.ObjectTracking },
+                LocationId = "us-east1"
```

Comment:
Add comment:
> It is recommended to use location_id as 'us-east1' for the best latency due to different types of processors used in this region and others.

From canonical. I never would've known this!

Unless the docs mention this above the sample. In which case it's not needed in the code (as this truth may change over time)

I do like having it in the code like the Python though